### PR TITLE
adding build script for myrepos

### PR DESCRIPTION
**Pull request by caseyjlaw**
Added a deployment script to test installation by myrepos.

### CODE REVIEW

**Comment by caseyjlaw** [comment]
Sorry, I added a commit here by mistake and it needs to get merged in. I'll do it myself, but comments on the "deploy" script are welcome.

**Comment by yupinghuang** [comment]
General comment tho - might be time for feature branches since PRs track branches, not commit. So like cjl/deploy or something. Also....Consider rebasing before PR and use rebase instead of merge. Merge can be quite untransparent and introduce unintended changes a times

File: deploy [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Installs contents of this repo.
+LOG=~/deploy_dsa110-pyutils.log
+DATE="`date --iso-8601='seconds'`"
+echo "$DATE: `which python`" > $LOG
+output=`pip install .`
```
[the remark this answers is not in the document]
Comment (yupinghuang) [reply deploy:7]: Also gonna be real annoying -- are we sure that every time this script gets run, the working directory is where you want it to be?

**Comment by caseyjlaw** [comment]
I copied this script from a repo by @rh-codebase. Any comment on these proposed changes?

**Comment by rh-codebase** [comment]
not sure where my comments go when I reply via e-mail. summary:

check command errors using $? not by output=`...` which captures stdout. If error, then it needs to be echoed so that the service can indicate back to the UI.

capture and log git describe as a sanity check. 

may not want to overwrite log file on each invocation.

**Comment by caseyjlaw** [comment]
There is something odd about conda environments setup with this script. When testing locally, it asks me to initialize the environment (`conda init <shell>`).
Running the script line by line executes as expected. I'm using pipeline user on cal-im in both cases.
Any ideas why conda treats this script like a fresh user?

**Pull request merged**